Add Example Queries for FAC API Calls (cURL, PowerShell, Python)

LynnMHouston opened the pull request:
Description:
As part of the API documentation update, I have added an examples.md page that provides sample API queries in multiple formats, including cURL, PowerShell, and Python.

Changes Included:
- Added examples.md
- Updated index.md to include examples page

Provided API call examples for:
 - cURL
 - PowerShell
 - Python (including pagination and filtering)
    
Testing Done:
- Verified correct page rendering and links.

jperson1 reviewed (approved):
Looks great to me! The API resources page is getting a little bloated, so perhaps we'll break things out more in the future. I like the quick collection of guides up top.

![image](https://github.com/user-attachments/assets/6de41cbc-e388-46ef-9777-29888240bc21)

jadudm commented:
I'd suggest at this part abandoning the cards, and going with a bulleted list and descriptions.

* Link: text
* Link: text
* ...

where the text is what we put on the front of the card.

LynnMHouston commented:
Thank you, @jperson1 and @jadudm! I look forward to implementing your suggestions in the next couple of weeks.

LynnMHouston commented:
Okay, I have updated the API index.md file to get rid of the cards and put in bullet points for what the cards used to cover. I updated the resources_page file to exclude the API landing page. Could you double-check that the examples.md page is properly linked in the bullets at the bottom of the index.md? I was counting on the examples page to have been approved before re-doing the index page, and I'm having a hard time visualizing.

LynnMHouston commented:
> I'd suggest at this part abandoning the cards, and going with a bulleted list and descriptions.
> 
> * Link: text
> * Link: text
> * ...
> 
> where the text is what we put on the front of the card.

Done!
